_data/simple-icons.json

```
@@ -11785,6 +11785,17 @@
             "hex": "0085DE",
             "source": "https://app.plangrid.com"
         },
+        {
+            "title": "Plasma",
+            "hex": "1D99F3",
+            "source": "https://kde.org/stuff/clipart/",
+            "guidelines": "https://kde.org/stuff/clipart/",
+            "aliases": {
+                "aka": [
+                    "KDE Plasma"
+                ]
+            }
```

```suggestion
            "title": "KDE Plasma",
            "hex": "1D99F3",
            "source": "https://kde.org/stuff/clipart/",
            "guidelines": "https://kde.org/stuff/clipart/"
```

I think we just call it KDE Plasma.